src/compiler/checker.ts

```diff
@@ -14308,6 +14310,10 @@ namespace ts {
             return (deferredGlobalBigIntType ||= getGlobalType("BigInt" as __String, /*arity*/ 0, /*reportErrors*/ false)) || emptyObjectType;
         }
 
+        function getGlobalRecordSymbol() {

```

Comment:
Part of me is not entirely comfortable long-term with using `Record` - especially with records and tuples. But maybe I can get past it.

Reply:
Well, `Record` isn't going anywhere given how much usage it has in existing code.